executor/write.go

```diff
@@ -1366,6 +1357,46 @@ func (e *UpdateExec) Next(goCtx goctx.Context) (Row, error) {
 	return Row{}, nil
 }
 
+// Next implements the Executor Next interface.
+func (e *UpdateExec) Next(goCtx goctx.Context) (Row, error) {
+	if !e.fetched {
+		err := e.fetchRows(goCtx)
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		e.fetched = true
+	}
+
+	return e.exec(goCtx, e.SelectExec.Schema())
+}
+
+// NextChunk implements the Executor NextChunk interface.
+func (e *UpdateExec) NextChunk(goCtx goctx.Context, chk *chunk.Chunk) error {
+	chk.Reset()
```

Comment:
for chunk execution, once `e.fetched == true` we can just `return nil`, because all the updating work is done in one `NextChunk()` function call.

Reply:
but return nil if e.fetched == true, it is wired.

Reply:
By doing this, if we call `NextChunk` multi-times, it will response quickly.

In fact, function `handleNoDelayExecutor()` should only call `NextChunk` or `Next` once, for some historical reasons it calls `Next` multi-times, but we can fix it to call `NextChunk` only once.